contracts/placeholder/test/public_api_placeholder.sol

```diff
@@ -20,6 +20,7 @@
 pragma solidity >=0.8.4;
 
 import "../../interfaces/verifier.sol";
+import "hardhat/console.sol";
```

Comment:
Can be removed.

Reply:
I removed it, of course.
But output with gas consumption is so much more informative!
It's so sad, that hardhat-gas-reporter is a bit useless for us now((